.github/workflows/jupyter-book.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 # This job installs dependencies, builds the book, and pushes it to `gh-pages`
 jobs:
-  deploy-book:
+  deployment:
```

Comment:
Just for my understanding, what does this change do?

Reply:
Nothing, just the titles of the workflows, e.g. pytest / testing (3.7) (pull_request) Successful in 38s or see this screenshot: 
<img width="860" alt="Screen Shot 2021-12-21 at 21 32 55" src="https://user-images.githubusercontent.com/67342040/147025234-aab14a6e-34a5-4ddf-9e49-97514de7f085.png">

Reply:
Note this particular one (jupyter-book) is only triggered if the /docs/ folder changes (after merging). These tests actually run twice: once here in the PR and once right after the PR, so the jupyter-book one is usually done only after

Reply:
Needless to say, I am happy to change the wording. I just wanted to make them as short and descriptive as possible